images/linux/scripts/helpers/invoke-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
```

Comment:
nitpick: I'd say we should simply the approach to invoke `invoke-test.sh` script and don't include `source $HELPER_SCRIPTS/invoke-tests.sh` statement in all bash scripts where we should invoke pester tests.

My proposal:
1. Convert `invoke_tests() {` function to script
2. Add execute permissions
3. Add a symlink to the one of well know paths from $PATH
4. Remove the symlink at the end of image generation process

Reply:
That is good idea!
I have logged separate work item on it: https://github.com/actions/virtual-environments-internal/issues/1616
To not block this PR for now